pekko-connectors-sample-jms/project/Dependencies.scala

```diff
@@ -1,25 +1,26 @@
 import sbt._
 
 object Dependencies {
-  val scalaVer = "2.13.7"
+  val scalaVer = "2.13.12"
   // #deps
-  val AkkaVersion = "2.6.19"
-  val AkkaHttpVersion = "10.1.12"
-  val AlpakkaVersion = "4.0.0"
+  val pekkoVersion = "1.0.1"
+  val pekkoHttpVersion = "1.0.0"
+  // todo: move to "1.0.2" after release in maven
+  val pekkoConnectorVersion = "1.0.1-RC1+6-dcc040bb-SNAPSHOT"
```

Comment:
pekkoConnectorVersion=1.0.2 is released - please do not use snapshot versions

Reply:
for JMS,there is released 1.0.1 in maven center(https://mvnrepository.com/artifact/org.apache.pekko/pekko-connectors-jms)

I will change it soon.